config/initializers/solidus_stripe.rb

```diff
@@ -0,0 +1 @@
+Stripe.api_key = Spree::PaymentMethod::StripeCreditCard.last&.preferences&.dig(:secret_key)
```

Comment:
The Stripe `secret_key` can also be defined in the database, which means it could be changed in the runtime. In that case, this would still have the old key. It would be better to define this during the runtime. This is also a good indication that we might need a class to handle the integration.